pallets/bridge-transfer/src/lib.rs

```diff
@@ -19,10 +19,13 @@
 
 pub use pallet::*;
 
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
 #[cfg(test)]
 mod mock;
 #[cfg(test)]
 mod tests;
+mod weights;
```

Comment:
I believe `weights` is actually not used. 
We need to specify the usage of it in extrinsic declaration, something like `[pallet::weight(<T as pallet::Config>::WeightInfo::....`

you could refer to other pallets